stl/inc/cmath

```diff
@@ -23,6 +23,7 @@
 #endif // _HAS_CMATH_INTRINSICS
 
 #if _HAS_CXX20
+#include <type_traits>

```

Comment:
`is_arithmetic_v` (and the `_Is_any_of_v` that I'm adding as well) is defined in `<xtr1common>` which is already included - we don't need `<type_traits>`.